pyproject.toml

```
@@ -1,94 +1,110 @@
-[build-system] # https://python-poetry.org/docs/pyproject/#poetry-and-pep-517
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
-
-[tool.poetry] # https://python-poetry.org/docs/pyproject/
+[project] # https://packaging.python.org/en/latest/specifications/pyproject-toml/
 name = "raglite"
 version = "0.6.2"
 description = "A Python toolkit for Retrieval-Augmented Generation (RAG) with SQLite or PostgreSQL."
-authors = ["Laurent Sorber <laurent@superlinear.eu>"]
 readme = "README.md"
-repository = "https://github.com/superlinear-ai/raglite"
-
-[tool.commitizen] # https://commitizen-tools.github.io/commitizen/config/
+authors = [
+  { name = "Laurent Sorber", email = "laurent@superlinear.eu" },
+]
+requires-python = ">=3.10,<4.0"
+dependencies = [
+  # Python:
+  # "python (>=3.10,<4.0)",
```

Would drop Python from the dependencies as it now has a dedicated `requires-python` attribute.
```suggestion
requires-python = ">=3.10,<4.0"
dependencies = [
```